tests/timetatbles/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Small nit but the `tests` directory path has a typo. Should be `tests/timetables/...`?

Reply:
Thanks! Fixed both.